go/grpcwebproxy/backend.go

```diff
@@ -48,6 +54,9 @@ func dialBackendOrFail() *grpc.ClientConn {
 	} else {
 		opt = append(opt, grpc.WithInsecure())
 	}
+	if *flagMaxMsgSize != 4194304 {
+		opt = append(opt, grpc.WithMaxMsgSize(flagMaxMsgSize))
```

Comment:
This call option is deprecated - please use `MaxCallRecvMsgSize` (https://github.com/grpc/grpc-go/blob/v1.8.2/rpc_util.go#L215).